src/main/java/com/uid2/admin/util/PrivateSiteUtil.java

```diff
@@ -139,6 +177,18 @@ private static <T> PrivateSiteDataMap<T> getPrivateSites(Collection<OperatorKey>
         return result;
     }
 
+    private static HashMap<Integer, Map<Integer, EncryptionKeyAcl>> getPrivateSiteAcls(Collection<OperatorKey> operators) {
+        HashMap<Integer, Map<Integer, EncryptionKeyAcl>> result = new HashMap<Integer, Map<Integer, EncryptionKeyAcl>>();
+        operators.forEach(o -> {
+            // TODO: Should we check if site is disabled?
```

Comment:
No, from talking to @atarassov-ttd  operator disabling is not implemented yet.

Reply:
Just leaving it here as a reminder